source/test/F0.Minesweeper.Components.Tests/MinefieldTests.cs

```diff
@@ -9,9 +13,20 @@ namespace F0.Minesweeper.Components.Tests
 {
 	public class MinefieldTests : TestContext
 	{
+		private readonly Mock<IMinefieldFactory> minefieldFactoryMock;
+		private readonly Mock<IMinefield> minefieldMock;
+
 		public MinefieldTests()
 		{
-			Services.AddSingleton(new Mock<ICellStatusManager>().Object);
+			minefieldFactoryMock = new Mock<IMinefieldFactory>(MockBehavior.Strict);
+			minefieldMock = new Mock<IMinefield>(MockBehavior.Strict);
+			Services.AddSingleton(minefieldFactoryMock.Object);
+			Services.AddSingleton<ICellStatusManager>(new CellStatusManager());
+		}
+
+		public override void Dispose()
+		{
+			Mock.VerifyAll(minefieldFactoryMock);
```

Comment:
[TestContextBase](https://github.com/egil/bUnit/blob/main/src/bunit.core/TestContextBase.cs) implements `Dispose` - we should call to the base-Dispose afterwards!